src/extension/gutters.ts

```diff
@@ -112,11 +111,11 @@ export class Gutters {
         this.outputChannel.appendLine(`[${Date.now()}][${area}]: ${message}`);
         this.outputChannel.appendLine(`[${Date.now()}][${area}]: ${stackTrace}`);
 
-        // Only send crash reports if the user allows this from their global settings.
-        const telemetry = workspace.getConfiguration("telemetry");
-        const enableCrashReporting = telemetry.get("enableCrashReporter");
-        if (enableCrashReporting) {
-            const sentryId = Sentry.captureException(error);
+        const crashReporter = new CrashReporter(false);
+
+        const sentryId = crashReporter.captureError(area, error);
+
+        if (sentryId) {
```

Comment:
we will want to move `crashReporter.captureError(area, error);` inside this if check to keep all the sentry bits in the same logically area.